qutip/tests/test_mesolve.py

```diff
@@ -893,7 +893,7 @@ def test_array_str_py_coeff(self):
             tlist=tlist, args={"_step_func_coeff": 1})
         result = mesolve(qu, rho0=rho0, tlist=tlist)
         assert(qu.type == "mixed_callable")
-        assert_allclose(
+        np.testing.assert_allclose(
```

Comment:
It might be better to remove this from the PR - it's not related to anything else you're doing here (nor to the test failure).  It's ok if you have a seemingly random test failure - one of the admins can check it over, and (as in this case) re-run the test if necessary.